native/src/main/java/io/ballerina/stdlib/crypto/nativeimpl/Sign.java

```diff
@@ -83,6 +83,12 @@ public static Object signRsaSha512(BArray inputValue, BMap<?, ?> privateKey) {
         return CryptoUtils.sign("SHA512withRSA", key, input);
     }
 
+    public static Object signRsaSsaPss256(BArray inputValue, BMap<?, ?> privateKey) {
+        byte[] input = inputValue.getBytes();
+        PrivateKey key = (PrivateKey) privateKey.getNativeData(Constants.NATIVE_DATA_PRIVATE_KEY);
+        return CryptoUtils.sign("SHA256withRSAandMGF1", key, input);
```

Comment:
Could you explain why we are using `SHA256withRSAandMGF1` here? Can't we use `SHA256withRSA/PSS` instead?